README.md

```diff
@@ -10,9 +10,12 @@ Franklin Dashboard allows you to check and manage your Franklin Projects.
     ```
     	GITHUB_CLIENT_ID=7047fds785d670dfsd9ab46    - App's Github Client ID
```

Comment:
That's what I figured. But since this is open source, we should make the instructions applicable to anyone so environment specific configuration should just be placeholders for now. Theoretically, anyone can land here and want so start contributing today.